### PR TITLE
Add link to project homepage

### PR DESCRIPTION
**Pull request by strk**
better than nothing, I think

### CODE REVIEW

**Review by tboerger — COMMENTED** [review]
And what's up with the sdk file?

**Comment by strk** [comment]
On Thu, Nov 10, 2016 at 09:11:27AM -0800, Thomas Boerger wrote:

> And what's up with the sdk file?

Pushed a link to the sdk with [gitea-page d9b02b8]

File: content/sdk.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ gosource: code.gitea.io/sdk https://github.com/go-gitea/gitea https://github.com
 ---
 
 # Go SDK
+
+This is the URL of the import path for the [Gitea](http://gitea.io) Go SDK
```
Comment (tboerger) [inline content/sdk.md]: Maybe we should directly link to the github repo instead of that. Same for gitea?

**Comment by strk** [comment]
Done, linked to github.

**Comment by tboerger** [comment]
LGTM

**Comment by andreynering** [comment]
LGTM

**Pull request merged**